server/swissvoice/static/css/style.css

```diff
@@ -4,6 +4,34 @@ body {
     padding: 0;
 }
 
+#bg-bubble {
+  background-image: url(../images/background/sprechblase.svg);
```

Comment:
while I do appreciate a good relative url, I do think that in this case we shouldn't use `/images/background/sprechblase.svg` (starting with a `/` means that the url is relative to the domain)

Reply:
Also, using a German filename all of a sudden doesn't really make sense.

Reply:
Didn't really know how to url
german filename will be changed